src/lib.rs

```diff
@@ -89,6 +99,46 @@ impl Deepgram {
     ///
     /// Panics under the same conditions as [`reqwest::Client::new`].
     pub fn new<K: AsRef<str>>(api_key: K) -> Self {
+        Self::with_base_url(api_key, DEEPGRAM_BASE_URL)
+    }
+
+    /// Construct a new Deepgram client with the specified base URL.
+    ///
+    /// When using a self-hosted instance of deepgram, this will be the
+    /// host portion of your own instance. For instance, if you would
+    /// query your deepgram instance at `http://deepgram.internal/v1/listen`,
+    /// the base_url will be `http://deepgram.internal`.
+    ///
+    /// Admin features, such as billing, usage, and key management will
+    /// still go through the hosted site at `https://api.deepgram.com`.
+    ///
+    /// Create your first API key on the [Deepgram Console][console].
+    ///
+    /// [console]: https://console.deepgram.com/
+    ///
+    /// # Example:
+    ///
+    /// ```
+    /// # use deepgram::Deepgram;
+    /// let deepgram = Deepgram::with_base_url(
+    ///     "apikey12345",
+    ///     "http://localhost:8080",
+    /// );
+    /// ```
+    ///
+    /// # Panics
+    ///
+    /// Panics under the same conditions as [`reqwest::Client::new`], or if `base_url`
+    /// is not a valid URL.
+    ///
+    ///
+
```

Comment:
Is there a reason for all this blank space here?

Reply:
Nope.